### PR TITLE
Adding support for EATON Accessory notification

### PR DESCRIPTION
**Pull request by r-teller**
Adding support for EATON Accessory notification

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/z-wave/device-specific.markdown [hunk 1]
```diff
@@ -1531,3 +1531,14 @@ Example Event:
     - service: switch.toggle
       entity_id: switch.office_fan
 ```
+
+### EATON On/Off & Dimmer (RF9501/RF9540-N/RF9640-N/RF9601-N)
```
Comment (frenck) [inline source/_docs/z-wave/device-specific.markdown:1535]:
```suggestion
### EATON On/Off & Dimmer (RF9501/RF9540-N/RF9640-N/RF9601-N)

```

**Comment by homeassistant** [comment]
Hi @r-teller, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16112).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_docs/z-wave/device-specific.markdown [hunk 2]
```diff
@@ -1531,3 +1531,14 @@ Example Event:
...
+Once you've added the remote to your Z-Wave network, you'll need to update your `zwcfg_*.xml` file with the below XML data. Stop Home Assistant and open your `zwcfg_*.xml` file (located in your configuration folder). Find the remote's device section and then its corresponding `CommandClass` section with id="112". Insert the snippet below into the CommandClass section with the below XML data. Save the file and restart Home Assistant.
+
+```xml
+				<Value type="list" genre="config" instance="1" index="10" label="Notify Accessory" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="1" vindex="1" size="1">
+					<Help>Ensures that changes to the master node automatically notify accessory switches</Help>
+					<Item label="disable" value="0" />
+					<Item label="enable" value="1" />
+				</Value>
```
Comment (frenck) [inline source/_docs/z-wave/device-specific.markdown]:
```suggestion
<Value type="list" genre="config" instance="1" index="10" label="Notify Accessory" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="1" vindex="1" size="1">
	<Help>Ensures that changes to the master node automatically notify accessory switches</Help>
	<Item label="disable" value="0" />
	<Item label="enable" value="1" />
</Value>
```

**Comment by homeassistant** [comment]
Hi @r-teller, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16112).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @r-teller 👍

**Pull request merged**